sphinx/api/configuration/metadata.rst

```diff
@@ -526,16 +526,6 @@ The metadata options for a configuration fall into four categories:
                     this % 2 == 1; # Needs to be odd
                     this * 3 > 100; # Needs to be more than 100/3.
 
-         A slightly different usage of this functionality can do things like
```

Comment:
This sentence is still necessary as it is a different usage.

Reply:
Ah I see. Sorry for attempting to change this - I thought it was simply a case of moving into the relevant new 'warn-if' section. Will retract.

Reply:
You were right to move this section but the "A slightly different usage of this functionality" sentence needs to go with it.

Reply:
I imagine replacing 'this' with 'the fail-if' or similar?

Reply:
Righto.